src/App.jsx

```diff
@@ -14,70 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const EMPTY_SPACE = '';
+const ADD_BUTTON_TEXT = '+';
+const REMOVE_BUTTON_TEXT = '-';
+const DEFAULT_GOOD_VALUE = 'Jam';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [goodState, setGoodState] = useState(DEFAULT_GOOD_VALUE);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const selectedGoodMessage = `${goodState} is selected`;
+  const noGoodSelectedMessage = 'No goods selected';
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {goodState.length ? (
+          <>
+            {selectedGoodMessage}
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
+              className="delete ml-3"
+              onClick={() => {
+                setGoodState(EMPTY_SPACE);
+              }}
+            />
+          </>
+        )
+          : noGoodSelectedMessage
+        }
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              key={good}
+              className={cn({
+                'has-background-success-light': good === goodState,
```

Comment:
You check the condition `good === goodState` several times, so it should be done once before returning the table component and using the finished result.